src/config/dynamic-theme-fixes.config

```diff
@@ -21293,6 +21293,45 @@ CSS
 
 ================================
 
+uottawa.brightspace.com
+
+CSS
+button:hover, button:focus, :host([active]) button {
+    --d2l-button-icon-background-color-hover: var(--darkreader-bg--d2l-color-gypsum);
+}
+.d2l-button-filter > ul > li > a.vui-button {
+    border-color: var(--darkreader-border--d2l-color-mica);
+}
+.d2l-label-text:has(.d2l-button-subtle-content):hover,
+.d2l-label-text:has(.d2l-button-subtle-content):focus,
+.d2l-label-text:has(.d2l-button-subtle-content):active {
+    background-color: var(--darkreader-bg-d2l-color-gypsum) !important;
+}
+.d2l-navigation-centerer {
+    color: inherit;
+}
+.d2l-navigation-link-image-container {
+    filter: invert(1);
+    mix-blend-mode: lighten;
+}
```

Comment:
Hmm this doesn't look sound, what does this specifically fix?

Reply:
The filter and mix-blend-mode is for the university logo in the nav bar. I tried using invert but it wouldn't look quite right.

However I did realize a problem with this (apart from the slight "jankiness" of it), which is that you can't see the logo if you set the scheme to light.

So I guess just remove it for now?

Fix with INVERT:

![WithINVERT](https://user-images.githubusercontent.com/48165845/217878536-14bc578c-f13b-4e22-8c86-ac2339d14d19.png)

Fix with CSS:

![WithCSS](https://user-images.githubusercontent.com/48165845/217878531-c8eb3e88-4e62-4348-a0ab-eaa258747cfe.png)

When using light scheme (when applying the CSS):

![WithCSS_Light](https://user-images.githubusercontent.com/48165845/217878533-61fe0200-ce43-4b4c-b26a-10084afed3b6.png)

Reply:
Yeah normally we just do `INVERT` and _accept_ the jankiness.

Reply:
Sorry by jankiness I was referring to my CSS patch. But ok, will just do invert.